lib/libc/minimal/source/string/string.c

```diff
@@ -116,6 +116,25 @@ size_t strlen(const char *s)
 	return n;
 }
 
+/**
+ *
+ * @brief Get fixed length string length
```

Comment:
This brief description does not make too much sense to me. 
How about "Determine the length of a fixed-size string"?

Or, to make it look similar to strlen, "Get fixed-size string length"

Reply:
Fixed.